src/modules/Interactive.js

```diff
@@ -0,0 +1,14 @@
+const ChangeTaskStatus = (arr, id) => {
```

Comment:
🎉🚀 You have done an excellent job with the implementation. However, I am currently experiencing some difficulties in updating your table. For example, I am able to add new tasks, but when I try to click on the checkbox to complete it, it functions properly. In your case we are able to update the localStore by clicking any area of the `list`.

However, the design is not showing me the updated action even after I reload the page. While we are able to make the incomplete action complete, we are not able to revert it back to its original state. Please refer to the video example provided for further clarification:

https://user-images.githubusercontent.com/69011963/213844026-b448231d-ba8a-4222-8ae4-b9414af5c70e.mov

- [ ] Please note that it is important for us to be able to update the table for both true/false actions for completed tasks. Thank you for your attention to this matter🙏.